packages/web/test/src/lib/web-mode.ts

```diff
@@ -59,22 +79,31 @@ describe("Webmode - Algosigner test cases ", function () {
 			const txnParams = await webMode.getSuggestedParams(execParams.payFlags);
 			const transactions: Transaction[] = webMode.makeTx([execParams], txnParams);
 			assert.doesNotThrow(async () => {
-				await webMode.signTx(transactions[0]);
+				const signTx = await webMode.signTx(transactions[0]);
+				expect(signTx).to.have.ownProperty("txn");
+				expect(signTx).to.have.ownProperty("sig");
+				expect(signTx.sig).to.exist;
 			});
 		});
 
-		it("Should return a SignedTransaction object based on ExecParams", async function () {
+		it("Should sign a lsig transaction and return an object with blob and txID", async function () {
 			const execParams: types.AlgoTransferParam = {
```

Comment:
why do we change an existing test here?

Reply:
because I updated the wallet mock as per the required return statement of `signTransaction` due to which this test become unusable